pdata/testdata/profile.go

```diff
@@ -22,6 +22,7 @@ func GenerateProfiles(profilesCount int) pprofile.Profiles {
 	ss := td.ResourceProfiles().At(0).ScopeProfiles().AppendEmpty().Profiles()
 
 	dic := td.ProfilesDictionary()
+	dic.StringTable().Append("")
```

Comment:
The next release of the proto will require default table entries for Mapping, Link and likely for Function. It might be a good idea to already set them up in this PR to reduce future noise.